library/blivet.py

```diff
@@ -88,20 +91,25 @@
     from blivet3.util import set_up_logging
     BLIVET_PACKAGE = 'blivet3'
 except ImportError:
-    from blivet import Blivet
-    from blivet.callbacks import callbacks
-    from blivet.flags import flags as blivet_flags
-    from blivet.formats import get_format
-    from blivet.partitioning import do_partitioning
-    from blivet.size import Size
-    from blivet.util import set_up_logging
-    BLIVET_PACKAGE = 'blivet'
-
-from ansible.module_utils.basic import AnsibleModule
-
-blivet_flags.debug = True
-set_up_logging()
-log = logging.getLogger(BLIVET_PACKAGE + ".ansible")
+    try:
+        from blivet import Blivet
+        from blivet.callbacks import callbacks
+        from blivet.flags import flags as blivet_flags
+        from blivet.formats import get_format
+        from blivet.partitioning import do_partitioning
+        from blivet.size import Size
+        from blivet.util import set_up_logging
+        BLIVET_PACKAGE = 'blivet'
+    except ImportError:
+        LIB_IMP_ERR = traceback.format_exc()
+
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
```

Comment:
This unfortunately won't work in Ansible 2.6 or 2.7 - `missing_required_lib` does not exist. Compare https://docs.ansible.com/ansible/2.7/dev_guide/developing_modules_best_practices.html#importing-and-using-shared-code and https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_best_practices.html#importing-and-using-shared-code